tmpl/blog/mirage-seal.md

```diff
@@ -26,47 +35,71 @@ Although there are many entities that can sign a certificate with different proc
 
 Log in to the web interface available through the registrar's website.  You can start the certificate signing process from the "services" tab, which exposes an "SSL" subtab.  Click that (Gandi doesn't need to know that we intend only to support TLS, not SSL).  Hit the "Get an SSL Certificate" button.  Standard SSL is fine.  Even if you're entitled to a free certificate, it will appear that you need to pay here; however at checkout, the total amount due will be 0 in your preferred currency.  Ask for a single address and, if you want to pay nothing, a valid period of 1 year.
 
 Copy the content of the certificate-signing request you generated earlier and paste it into the web form.  Gandi will also ask you to identify your TLS stack; unfortunately `ocaml-tls` isn't in the drop-down menu, so choose OTHER (and perhaps send them a nice note asking them to add the hottest TLS stack on the block to their list).  Click "submit" and click through the order form.
 
-From here on out, the process is automatic if you're buying a certificate for a domain you have registered through them, via the account with which you requested the certificate.  You should shortly receive an e-mail with a subject like "Procedure for the validation of your Standard SSL certificate", which explains the process in more detail, but really all you need to do is wait a while (about 30 minutes, for me).  After the certificate has been generated, Gandi will notify you by e-mail, and you can download your certificate from the SSL management screen.  Click the magnifying glass next to the name of the domain for which you generated the cert to do so.
+If you're buying a certificate for a domain you have registered through Gandi (via the registered account), the rest of the process is pretty automatic.  You should shortly receive an e-mail with a subject like "Procedure for the validation of your Standard SSL certificate", which explains the process in more detail, but really all you need to do is wait a while (about 30 minutes, for me).  After the certificate has been generated, Gandi will notify you by e-mail, and you can download your certificate from the SSL management screen.  Click the magnifying glass next to the name of the domain for which you generated the cert to do so.
+
+Once you've downloaded your certificate, you may also wish to append the [intermediate certificates](https://en.wikipedia.org/wiki/Intermediate_certificate_authorities).  Here's a help page on [gathering intermediate certificates](https://wiki.gandi.net/en/ssl/intermediate).  Equipped with the intermediate certificates, append them to the signed certificate downloaded for your site to provide a full certificate chain:
+
+```
+cat signed_cert.pem intermediate_certs.pem > server.pem
+```
+
+#### Example: StartSSL.com
```

Comment:
I'm too bad at counting, but the four `####` should be five `#####` (as above with gandi).